### DISCUSSION TITLE
Does tauri support java interface

### DISCUSSION THREAD

**Discussion by hanxiaohuivc** [tauri-apps/tauri#2413] [Q&A] [answered]
Hello everyone. I have a project that includes front-end (the front-end is a web page written by html, css, js) and back-end (back-end is a business system written in java）, and the front and back ends use ‘ajax’ for interaction. If I use tauri to package the front-end files, does the interaction between the app.exe and the back-end need to be rewritten using tauri API?
When I use the "yarn tauri dev" command to debug, I can log in app.exe to view database information and other operations. However, when I run the exe generated by "yarn tauri build", I cannot log in.

**Comment by amrbashir** [comment]
related to #2327

**Comment by hanxiaohuivc** [comment, marked as answer]
I successfully tested tauri to access webapi. Thank you.

**Reply by whatkind** [reply]
将vue发到tauri上打包后的后台请求地址应该是http://ip:port/xxx，为啥打包后变成了http://tauri.localhost:port/xxx
